### ISSUE TITLE
netdata-installer.sh uses the wrong installation directories on FreeBSD.

### ISSUE THREAD

**Issue by vlvkobal** [netdata/netdata#4303]
File system hierarchy in FreeBSD is slightly different than in Linux. Hardcoded `/usr/sbin`, `/usr/share`, `/usr/libexec` like in https://github.com/netdata/netdata/blob/6ec160563b988e035f0f9e9e096fcff60b924ba0/netdata-installer.sh#L294 should be changed.

From [hier(7)](https://www.freebsd.org/cgi/man.cgi?query=hier&apropos=0&sektion=0&manpath=FreeBSD+11.2-RELEASE+and+Ports&arch=default&format=html):
```
local/	local	executables, libraries,	etc.  Also used	as the
	default destination for the ports(7) framework.
	Within local/, the general layout sketched out by
	hier for /usr	should be used.	 Exceptions are	the
	man directory	(directly under	local/ rather than
	under	local/share/), ports documentation (in
	share/doc/_port_/), and /usr/local/etc (mimics
	/etc).
```
@mmokhi, could you provide the list of the correct directories for the netdata installation on FreeBSD. Is it the same for installation from sources, ports, and packages?
@rickard-von-essen, what is the installation practice for macOS?

**Comment by rickard-von-essen** [comment]
For the macOS homebrew install we use `configure --prefix=<> --sysconfdir=<>; make install` to install the `netdata-installer.sh` is not used. (By default Homebrew installs packages in `/usr/local/Cellar/netdata/<version>/`.)

**Comment by rickard-von-essen** [comment]
The [FreeBSD port](https://svnweb.freebsd.org/ports?view=revision&revision=465801) installs everything correctly and doesn't either use `netdata-install.sh`.

**Comment by openspork** [comment]
I would like to see this implemented as well -- below is where the FreeBSD port places components:
root@znas:/usr/local # find . -iname netdata -exec realpath {} \;
/usr/local/share/netdata        #web resources
/usr/local/etc/netdata            #configuration
/usr/local/etc/rc.d/netdata    #service definition
/usr/local/sbin/netdata          #binary
/usr/local/libexec/netdata     #executable(script or binary) plugins/modules

**Comment by cakrit** [comment]
Added label feature request, so it doesn't go stale again.

**Comment by Ferroin** [comment]
I'll be looking into getting this fixed after finishing #7248 and #7585.

**Comment by Ferroin** [comment]
This obviously did not get tackled back when I intended to. As of right now, it’s planned to be tackled as part of the overhaul of the installer/updater code.
